src/markermedia/NFTaddTJS.ts

```diff
@@ -322,6 +322,11 @@ export default class NFTaddTJS {
         this.names.push(name);
     }
 
+    public remove(name: string) {
```

Comment:
i think this function should be renamed removeRoot because actually it does exactly this.